daemons/validation-scheduler-daemon/Makefile

```diff
@@ -0,0 +1,39 @@
+include ../../common.mk
+.PHONY: install build stage deploy clobber
+
+ZIP_FILE=validation_scheduler_daemon.zip
+BUCKET=$(BUCKET_NAME_PREFIX)lambda-deployment-$(DEPLOYMENT_STAGE)
+STAGED_FILE_KEY=$(ZIP_FILE)
+
+default: build
+
```

Comment:
Maybe not in this pr, but at some point we should combine the makefiles for all the lambdas. I think we could put everything other than deploy and the zip_file name in common.mk

Reply:
Great idea! Will create a ticket